tls_main.c

```diff
@@ -908,13 +949,21 @@ static int __init tls_register(void)
 	tls_device_init();
 	tcp_register_ulp(&tcp_tls_ulp_ops);
 
+    L = luaL_newstate();
```

Comment:
Not ideal.
This will share one Lua state between all processes that use TLS.
The mutex access will pay a huge price in a multi-threaded web server.
Also, this would share the scripts between the processes.
What if there are two scripts that define two different functions but with the same name?